NetscapeBookmarkParser.php

```diff
@@ -37,6 +51,15 @@ public function __construct(
             $this->defaultTags = array();
         }
         $this->defaultPub = $defaultPub;
+
+        $logsDir = ! empty($logsDir) ? $logsDir : __DIR__ . '/logs/';
+        $this->logger = new Logger(
+            $logsDir,
+            $logLevel,
+            array(
+                'filename' => 'import.log',
```

Comment:
- could be parameterized if we want to append logs to an existing file
- the default filename should include the current timestamp to avoid being overriden and keep one log file per imported bookmark file